### PR TITLE
Desktop version (third review)

### PR DESCRIPTION
**Pull request by utsargo**
This is the desktop version of the site. Implemented media queries for responsiveness. All codes are refactored for desktop compatibility. 

- Target viewport: 1440px

After getting the first review, I made the following changes.

- Fixed double navigation menu issue on a certain viewport.
- Changed the style of typography in the menu.
- Adjusted gaps between the sections according to figma design.
- Changed button colors according to figma design.
- Added an hr element in the footer for desktop.
- Added cursor pointer property to the contact form buttons.
- Changed the element of class 'wrapper' from 'div' to 'main'.
- Changed the double H1 from the welcome section to single one.

**Changes made after 2nd review:**
- Resolved navigation menu conflict

### CODE REVIEW

File: assets/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+  .navbar .desktop-nav ul li a:hover {
+    color: #36b37e;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .desktop-form {
+    display: none;
+  }
+}
```
Comment (Gopxfs) [inline assets/styles/style.css]: Kindly notice that your navbar currently has a breakpoint that is displaying both desktop & mobile versions 😉:
image

To fix this, you could add the following styling to your media query (max width: 768px)
```css
.desktop-nav {
  display: none
}
```

File: assets/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .navbar .desktop-nav ul li a {
+    padding: 10px;
+    text-decoration: none;
+    color: #172b4d;
+  }
```
Comment (Gopxfs) [inline assets/styles/style.css]: Kindly notice that the desktop `nav` elements have a font-weight of 600 and a different color 😉:

| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

> Tip: you can always select any element on [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A2068) to check the exact properties 😄.

File: assets/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+  object-fit: cover;
+  min-height: 648px;
+  padding: 64px 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
```
Comment (Gopxfs) [inline assets/styles/style.css:145]: Please give more spacing between your `welcome` section and the `recent-works`, as displayed on the original design 😀:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

File: assets/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,514 @@
...
+  background-color: #36b37e;
+  color: white;
+  transition: 500ms ease;
+}
+
+.btn-primary:hover {
+  background-color: #ebf0ee;
+  border: 2px solid #505f79;
+  color: #172b4d;
+}
```
Comment (Gopxfs) [inline assets/styles/style.css]: Awesome job adding a hover effect to your buttons 😄! Although, please use the same color scheme as the ones displayed in Figma:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.

File: assets/styles/style.css [hunk 5]
```diff
@@ -0,0 +1,514 @@
...
+  background-color: #36b37e;
+}
+
+/* section recent-works */
+.recent-works {
+  padding: 64px 24px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (Gopxfs) [inline assets/styles/style.css:247]: Kindly add more margin between your `recent-works` and `about-me` sections, as displayed in Figma 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

File: assets/styles/style.css [hunk 6]
```diff
@@ -0,0 +1,514 @@
...
+  }
+
+  .contact-me .section-heading {
+    font-size: 30px;
+  }
+
+  .social-links-footer {
+    border-top: 1px solid #505f79;
+    padding: 5px;
+  }
```
Comment (Gopxfs) [inline assets/styles/style.css]: Kindly adjust the color of the footer line, as displayed in Figma 😉:

| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

Also, please notice that this is a different element, not a border-top from the social links.
Tip: you can use a `hr` element 😄.

File: index.html [hunk 7]
```diff
@@ -164,22 +204,47 @@ <h3 class="card-title">
...
+                method="POST"
+                >
+                    <fieldset>
+                    <input type="text" name="first-name" maxlength="30" placeholder="First name" required>
+                    <input type="text" name="last-name" maxlength="30" placeholder="Last name" required>
+                    </fieldset>
+                    <input type="email" name="email" placeholder="Email Adress" required>
+                    <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+                    <button class="button btn-primary border-none" type="submit">Get In touch</button>
+                </form>
```
Comment (Gopxfs) [inline index.html:235]: [OPTIONAL] Kindly notice that hovering over your form button doesn't change the cursor to pointer. If you want, you can add this effect:
```css
.button {
    cursor: pointer;
}
```

File: index.html [hunk 8]
```diff
@@ -5,50 +5,57 @@
...
-                    <div class="portfolio-text display-flex flex-column txt-center align-middle">
+                    <div class="portfolio-text">
                         <h3 class="card-title">
                             Multi-Post Stories Gain+Glory
                         </h3>
-                        <ul class="categories display-flex align-middle justify-center">
+                        <ul class="categories">
                             <li><a href="" class="category">Ruby on rails</a></li>
                             <li><a href="" class="category">css</a></li>
                             <li><a href="" class="category">Javascript</a></li>
```
Comment (Gopxfs) [inline index.html]: Please notice that these elements shouldn't be `a` tags 😉. As displayed in Figma:
image

You can still keep the hover effect, it looks very cool 😀.

> Note: you did an impressive job with the design! Kudos to you, well done 😄! Keep up the good work, you will be an amazing and very talented developer, I'm sure 😉.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @utsargo 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Followed the design almost perfectly!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/styles/style.css [hunk 9]
```diff
@@ -0,0 +1,604 @@
...
+
+  hr {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .wrapper {
+    max-width: 768px;
+  }
```
Comment (emmiiorji) [inline assets/styles/style.css:445]: You have done great adjusting the breakpoints of the mobile menu as requested by the previous reviewer. However, there still seems to be an overlap for screen sizes greater than 768px and less than 1024px. Kindly look into that too.


image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @utsargo, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Good Highlights✔

- Linter checks are passing
- Good commit messages
- Adherence to GitHub flow
- Some requested changes have been implemented


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/styles/style.css [hunk 10]
```diff
@@ -0,0 +1,604 @@
...
+
+.down-arrow:hover {
+  color: white;
+  background-color: #36b37f;
+}
+
+/* section recent-works */
+.recent-works {
+  padding: 64px 24px;
+  display: flex;
```
Comment (emmiiorji) [inline assets/styles/style.css:244]: [Optional] I recommend that you adjust the margin of the cards to correspond to the design by figma.

Your design:
image


Figma design:
image

**Review by ichala — APPROVED** [review]
Hi @utsargo ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**